ee-feature-pack/common/src/main/resources/modules/system/layers/base/org/wildfly/clustering/infinispan/spi/main/module.xml

```diff
@@ -46,5 +46,6 @@
         <module name="org.wildfly.clustering.marshalling.spi"/>
         <module name="org.wildfly.clustering.service"/>
         <module name="org.wildfly.security.elytron-private"/>
+        <module name="org.wildfly.common"/>
```

Comment:
Nit: Please retain the lexicographical ordering.

Reply:
I'll fix this later.

Reply:
@pferraro Sorry, I did not recognize you meant the existing entries also. Next time better. Thanks for taking the time to review and merge.